src/ZoneServer/World/Actors/Monsters/Mob.cs

```diff
@@ -307,11 +316,31 @@ public void Kill(ICombatEntity killer)
 
 			this.DisappearTime = DateTime.Now.AddSeconds(2);
 
-			if (killer is Character characterKiller)
+			// If I have a master, remove my variables from my master so I can be resummoned
+			if (this.Components.Get<AiComponent>()?.Script.GetMaster() != null)
+			{
+				if (this.Components.Get<AiComponent>()?.Script.GetMaster() is Character master)
```

Comment:
This is another instance where you get the master over and over, here as well as below.

Reply:
This is fixed.